docs/shared/key_path.rst

```diff
@@ -0,0 +1,11 @@
+The ``--key-path`` option can be used to navigate from the top of the config to the driver's configuration block. For example, specifying ``--key-path foo.bar`` with config
+
+  .. highlight:: yaml
+  .. literalinclude:: ../../../../shared/key_path_nested.yaml
```

Comment:
Failing the CI/CD run with an error: 
```
Warning, treated as error:
/Users/naureenbharwani/Desktop/UFS/uwtools/docs/shared/key_path.rst:6:Include file '/Users/naureenbharwani/Desktop/shared/key_path_nested.yaml' not found or reading it failed
```

The documentation is being generated correctly [here](https://uwtools--490.org.readthedocs.build/en/490/sections/user_guide/cli/drivers/chgres_cube.html).

Reply:
I checked out your branch and had a look at this and think I know the solution. Try specifying
```
.. literalinclude:: /shared/key_path_nested.yaml
```
and
```
.. literalinclude:: /shared/key_path_simple.yaml
```
Apparently, and this was news to me, Sphinx interprets absolute paths as relative to "the top source directory" which, in our case, for docs, is `docs/`. So, for me, the above works.

Also, I think I misled you about the indentation: I think the lines in this file should be all the way to the left, but the line
```
.. include:: ../../../../shared/key_path.rst
```
in `chgres_cube.rst` should be indented to match its context.